### PR TITLE
Pydantic models config field

### PR DESCRIPTION
**Pull request by danicc097**
Since dataclasses are the default since #939, I added an optional `use_pydantic_models` field to use Pydantic for all models (discussed since #923) which requires much less boilerplate later at least for my use case

### CODE REVIEW

File: protos/plugin/codegen.proto [hunk 1]
```diff
@@ -68,6 +68,7 @@ message PythonCode
   bool emit_async_querier = 3;
   string package = 4;
   string out = 5;
+  bool use_pydantic_models = 6;
```
Comment (kyleconroy) [inline protos/plugin/codegen.proto:71]: To match the other configuration options, rename this to `emit_pydantic_models`.

File: examples/python/requirements.txt [hunk 2]
```diff
@@ -3,3 +3,4 @@ pytest-asyncio~=0.14.0
 psycopg2-binary~=2.8.6
 asyncpg~=0.21.0
 sqlalchemy==1.4.0
+pydantic~=1.9.0
```
Comment (kyleconroy) [inline examples/python/requirements.txt:6]: remove

File: internal/plugin/codegen.pb.go [hunk 3]
```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.27.1
-// 	protoc        v3.19.4
+// 	protoc-gen-go v1.26.0
+// 	protoc        v3.6.1
```
Comment (kyleconroy) [inline internal/plugin/codegen.pb.go:4]: Please install the same versions of protoc and protoc-gen-go.

File: examples/python/sqlc.json [hunk 4]
```diff
@@ -10,7 +10,8 @@
           "out": "src/authors",
           "package": "authors",
           "emit_sync_querier": true,
-          "emit_async_querier": true
+          "emit_async_querier": true,
+          "use_pydantic_models": true
```
Comment (kyleconroy) [inline examples/python/sqlc.json:14]: Instead of updating the example, please add a new end-to-end test for Pydantic models.

**Review by kyleconroy — CHANGES_REQUESTED** [review]
Thanks for the first draft. Requested a few changes; let me know if you have any questions.

File: examples/python/requirements.txt (hunk 2 above)
Comment (danicc097) [reply examples/python/requirements.txt:6]: isn't this required for ci to pass?

**Review by kyleconroy — APPROVED** [review]

**Pull request merged**